src/cmd/yacc.c

```diff
@@ -1177,19 +1177,23 @@ setup(int argc, char *argv[])
 {
 	long c, t;
 	int i, j, fd, lev, ty, ytab, *p;
-	int vflag, dflag, stem;
+	int vflag, dflag, fdflag, stem;
 	char actnm[8], *stemc, *s, dirbuf[128];
 	Biobuf *fout;
 
 	ytab = 0;
 	vflag = 0;
 	dflag = 0;
+	fdflag = 0 != 0;
 	stem = 0;
 	stemc = "y";
 	foutput = 0;
 	fdefine = 0;
 	fdebug = 0;
 	ARGBEGIN{
+	case 'f':
```

Comment:
Intuitively, I would think that `-f` would take a filename, not a file descriptor. Consider renaming this argument: perhaps `-F` would be sufficiently distinct?

Reply:
Though on further thought, file descriptors on plan9 appear in a filesystem namespace; under plan9, one wouldn't bother with this but would rather use some file name for an arbitrary FD: `/fd/12` for example.

On at least Linux, FreeBSD, OpenBSD and macOS, `/dev/fd` is contains files that seems to perform roughly the same function as on plan9. Given that those exist, one wonders whether this particular change is necessary; in the minimalist spirit of plan9 and its ports, I'd suggest that it is not.

Reply:
@dancrossnyc I rm'd that commit now.